flink-python/pyflink/common/sql_dialect.py

```diff
@@ -24,19 +24,21 @@ class SqlDialect(object):
     """
     Enumeration of valid SQL compatibility modes.
 
-    For most of the cases, the built-in compatibility mode will suffice. For some features,
+    In most of the cases, the built-in compatibility mode will suffice. For some features,
     i.e. the "create partitionable table" grammar is only supported in Hive dialect, you may need
```

Comment:
I just checked the Hive documentation. Such a statement does not exist in Hive, is that correct?

Reply:
For "create partitionable table" i actually mean create a partitionable table, it is not really a sql grammar, maybe i should remove the quotes.

Reply:
Thanks for the clarification @danny0405. I thought you mean some grammar. I will merge this PR now.